salt/pillar/nsot.py

```diff
@@ -92,7 +92,9 @@ def _get_token(url, email, secret_key):
                                   decode=True)
     error = query.get('error')
     if error:
-        log.error('can\'t get auth_token from nsot! reason: %s', error)
+        log.error('nsot ext pillar: can\'t get auth_token! reason: %s', error)
+        log.error('nsot ext pillar: verify that %s is reachable', url)
+        log.error('nsot ext pillar: verify that %s is a valid email', email)
```

Comment:
OK, but we don't do it that way normally. How about this:

```python
log.error('Cannot obtain NSoT authentication token due to: %s.', error)
log.debug("Please verify NSoT URL '%s' is reachable and email is valid", url, email)
```

Reply:
This would put me over the 80 char line limit. I'm struggling to find a way to do this that fits within that constraint. Here's what I have:

```
if error:
        log.error('Cannot obtain NSoT authentication token due to: %s.', error)
        log.debug('verify NSoT URL %s is reachable', url)
        log.debug('verify email %s is valid', email)
```

Reply:
Great! Thanks.